x509/src/lib.rs

```diff
@@ -14,6 +14,8 @@ extern crate alloc;
 #[cfg(feature = "std")]
 extern crate std;
 
+pub use ::der;
```

Comment:
Unless it's actually ambiguous, I'd suggest just using:

```suggestion
pub use der;
```

...otherwise it renders like this in the rustdoc:

<img width="226" alt="Screen Shot 2022-03-07 at 11 29 48 AM" src="https://user-images.githubusercontent.com/797/157095626-2fb6e823-6ce9-4dd8-ad38-4b239e6ef04c.png">

Reply:
Fixed.